src/vivarium_public_health/disease/special_disease.py

```diff
@@ -253,6 +269,11 @@ def continuous_filter(index):
 
         return filter_function
 
+    def adjust_state_and_transitions(self):
```

Comment:
Ah, see you're doing a thing here.  If the disease is recoverable, you have an SIS model, not an SIR model.  If the disease is not recoverable you have an SI model.